### PR TITLE
[MAJOR] Created the sidebar and added it to the relevant pages.

### LINKED ISSUE #57: Create side bar

### PR DESCRIPTION
**Pull request by jmezaber**
closes #57 

Sidebar demo
Sidebar demo - responsive

### CODE REVIEW

File: frontend/src/App.tsx [hunk 1]
```diff
@@ -21,16 +23,102 @@ const App = () => {
...
+                    <div style={{ paddingTop: '75px' }}>
+                      <Sidebar />
+                      <CreateEmployee />
+                    </div>
+                  </React.Fragment>
+                )}
+              />
               <Route exact path="/login" render={() => <Login />} />
-              <Route exact path="/" render={() => <div>root</div>} />
+              <Route
```
Comment (nasaku898) [inline frontend/src/App.tsx:113]: I don't think the root should have the side bar.

File: frontend/src/components/Sidebar/Sidebar.tsx [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+      onClick: () => history.push('/clientAccount/new'),
+    },
+    {
+      text: 'Create Employee Account',
+      icon: <CreateIcon className={classes.icon} />,
+      onClick: () => history.push('/employeeAccount/new'),
+    },
+  ];
+
+  function handleClick() {
```
Comment (nasaku898) [inline frontend/src/components/Sidebar/Sidebar.tsx:72]: use arrow function instead of functions.

**Review by nasaku898 — CHANGES_REQUESTED** [review]
Looks good, but I added some minor comments to change.

File: frontend/src/components/Sidebar/Sidebar.tsx [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+            </ListItem>
+          );
+        })}
+      </List>
+    </Box>
+  );
+
+  return (
+    <div>
+      {(['left'] as const).map((anchor) => (
```
Comment (nasaku898) [inline frontend/src/components/Sidebar/Sidebar.tsx:99]: Why do you iterate through this array when there is only 1 hard coded element?

**Review by Manraj45 — CHANGES_REQUESTED** [review]
Could you add the logout button?

File: frontend/src/App.tsx (hunk 1 above)
Comment (jmezaber) [reply frontend/src/App.tsx]: It has been removed!

File: frontend/src/components/Sidebar/Sidebar.tsx (hunk 2 above)
Comment (jmezaber) [reply frontend/src/components/Sidebar/Sidebar.tsx]: Done!

File: frontend/src/components/Sidebar/Sidebar.tsx (hunk 3 above)
Comment (jmezaber) [reply frontend/src/components/Sidebar/Sidebar.tsx]: I removed the iteration, thank you!

**Review by nasaku898 — APPROVED** [review]
Looks great!

**Review by Manraj45 — APPROVED** [review]
Nice work! We will need to handle the logout error for next sprint.

**Pull request merged**